### ISSUE TITLE
Remove unnecessary spaces in hompage ui

### ISSUE THREAD

**Issue by jitiendran** [arclix/arclix-docs#31]
### Description

User shouldn't waste most of their time scrolling through the website and also since it is homepage it should be short and crisp

### Suggested solution

Remove unnecessary spaces from the hom page UI

### Validations

- [X] Follow our [Code of Conduct](https://github.com/arclix/arclix-docs/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/arclix/arclix-docs/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://arclix.github.io/arclix-docs/docs/intro).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by jitiendran** [comment]
[ Triaged ] Good to start the development
